projects/SynthIPRuntime/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_executable(Runtime main.cpp)
+
+target_compile_options(Runtime PUBLIC)
```

Comment:
Last commit message should be adapted: "Exclude exceptions for all case study except this one :wink: "